packages/cli/next-hashicorp

```diff
@@ -19,7 +19,7 @@ const {
 } = require('./manifest')
 const generators = require('./generators')
 const open = require('open')
-const datoApiConfig = require('@hashicorp/platform-cms/config')
+const datoApiConfig = require('@hashicorp/platform-cms/config.cjs')
```

Comment:
Exporting a cjs version of `platform-cms` to avoid a larger refactor for this CLI package.